runtime/queries/metricsview_timeseries.go

```diff
@@ -109,13 +110,13 @@ func (q *MetricsViewTimeSeries) buildMetricsTimeSeriesSQL(mv *runtimev1.MetricsV
 
 	whereClause := "1=1"
 	args := []any{}
-	if mv.TimeDimension != "" {
+	if timestampColumnName != "" {
```

Comment:
This check will always pass since `timestampColumnName` is now quoted – better to keep using `mv.TimeDimension` for the check